examples/cpp/Makefile

```diff
@@ -46,5 +46,8 @@ elliptic1D: elliptic1D.cpp
 elliptic2D: elliptic2D.cpp
 	$(CXX) $(CXXFLAGS) $(INCPATH) -o elliptic2D elliptic2D.cpp $(LIBS)
 
+Schrodinger2D: Schrodinger2D.cpp	
+	$(CXX) $(CXXFLAGS) $(INCPATH) -o Schrodinger2D Schrodinger2D.cpp $(LIBS)
```

Comment:
Hi, @arrowguy234 , please edit the name of the executable or the name of the file you intend to push to the repository. At the moment, they do not match. Your file is named `Schrodinger2D-2.cpp` and in this Makefile you are telling the compile to compile the rule `Schrodinger2D`. If you test this, by running 
```
make
```
you will incur in the following error message:

```
make: *** No rule to make target 'Schrodinger2D.cpp', needed by 'Schrodinger2D'.  Stop.
```
Between the two, I would personally prefer that you rename your file `Schrodinger2D-2.cpp` ->  `Schrodinger2D.cpp`. So you would not have to modify all the instances in the Makefile

Reply:
Sorry for the mistake, having 2 testing files resulted in this error, i have changed the file name